modules/client-configuration/pages/configuration-management.adoc

```diff
@@ -166,20 +176,29 @@ If the client's configuration priorities are going to be reordered by an action,
 
 
 
-== Compare Configuration Files
+== Compare Configuration Files
 
 You can also use the system set manager (SSM) to compare the configuration file deployed on clients with the configuration file stored on the {productname} Server.
 
 
 
-.Procedure: Comparing Configuration Files
+.Procedure: Comparing Configuration Files
 . In the {productname} {webui}, navigate to menu:Systems[Systems List] and select the clients subscribed to the configuration files you want to compare.
 . Navigate to menu:Systems[System Set Manager], and go to the menu:Configuration[Compare Files] subtab to the list of available configuration files.
 . OPTIONAL: Click the number in the [guimenu]``Systems`` column to see which clients are currently subscribed to the configuration file.
 . Check the configuration files to compare, and click btn:[Schedule File Comparison].
 
 
 
+== Jinja templating on {salt} Clients
+
+Jinja templating is possible on {salt} Clients.
+Jinja variables can be used in {salt} pillars or grains in a configuration file of a {salt} state channel.
```

Comment:
I do not understand this sentence. This does not sound correct.

Do you mean something like this?

Jinja provide variables (made) from pillars or grains. They can be used in configuration files or salt states.